static/script/devices/mediaplayer/samsung_streaming.js

```diff
@@ -386,6 +379,16 @@ require.def(
             getSeekableRange: function () {
                 return this._range;
```

Comment:
samsing_maple.js in this instance isnt exhibiting best practices, however its approach is acceptable in the maple world as there is no live seeking. We **require this to be changed** to prevent future anomalies/issues. Please look at the cehtml.js modifier for an example of this that is proven and works.